presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetWriterConfig.java

```diff
@@ -31,7 +32,8 @@ public DataSize getBlockSize()
         return blockSize;
     }
 
-    @Config("hive.parquet.writer.block-size")
+    @Config("parquet.writer.block-size")
+    @LegacyConfig("hive.parquet.writer.block-size")
```

Comment:
instead of adding `@LegacyConfig` you can bind the config as `configBinder(binder).bindConfig(ParquetWriterConfig.class, "hive")` in Hive

Reply:
Oh, I see. But won't that break any configurations using the current property name for Iceberg?

Reply:
I think it will and i don't think we can avoid it. I understand this is what @electrum was after in https://github.com/prestosql/presto/pull/5376#pullrequestreview-500411843
Still, the connector was officially added a month ago, so I hope this is not a big problem.